label.py

```diff
@@ -1,11 +1,17 @@
-import csv
 from PIL import ImageTk, Image
 from tkinter import Tk, Label
+import argparse
```

Comment:
Nit: move this at the top. For imports we're using the following ordering:
1. Built-in modules first
2. Modules from third-parties
3. Our modules (like `utils`)

Reply:
Done!